packages/kobber-components/src/checkbox/checkbox.styles.ts

```diff
@@ -0,0 +1,90 @@
+import { css } from "lit";
```

Comment:
Burde ikke dette være i en css-fil, istedet for ts? Og hvis vi bruker CSS Modules slipper vi lange BEM-selectorer.

Reply:
Klarer ikke helt å finne noe god dokumentasjon på hvordan man bruker eksterne css-filer med lit så dropper det i denne omgang, men kult hvis du får det til @olabaloo. 
Noen på stachoverflow som sier det er en dårlig idé her hvertfall... https://stackoverflow.com/questions/69613048/how-to-load-external-css-file-with-lit